src/components/Attachments/AttachmentView/index.js

```diff
@@ -160,8 +160,9 @@ function AttachmentView({
         const encryptedSourceUrl = isAuthTokenRequired ? addEncryptedAuthTokenToURL(source) : source;
 
         const onPDFLoadComplete = (path) => {
-            if (path && (transaction.transactionID || reportActionID)) {
-                CachedPDFPaths.add(transaction.transactionID || reportActionID, path);
+            const id = (transaction && transaction.transactionID) || reportActionID;
+            if (path && id) {
+                CachedPDFPaths.add(id, path);
```

Comment:
@tienifr just noticed this, since we should not be storing the path on web could we only add it on native?

Reply:
>CachePDFPath already has platform-dependent files (i.e. noop functions on web).

Forgot about that 😆